### PR TITLE
BAH-2626 | Upgraded dependencies

### PR DESCRIPTION
**Pull request by SanoferSameera**
- Spring boot 2.7.6
- log4j 2.17.2
- postgresql 42.5.0
- atomfeed-client 1.10.1

### CODE REVIEW

File: hip-atomfeed/pom.xml [hunk 1]
```diff
@@ -69,6 +81,7 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <version>2.7.1</version>
```
[the remark this answers is not in the document]
Comment (kavitha-sundararajan) [reply hip-atomfeed/pom.xml]: refactored

File: pom.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.2.4.RELEASE</version>
+        <version>2.7.6</version>
```
[the remark this answers is not in the document]
Comment (kavitha-sundararajan) [reply pom.xml:15]: refactored

File: hip-atomfeed/pom.xml [hunk 3]
```diff
@@ -134,7 +130,7 @@
         <dependency>
             <groupId>org.liquibase.ext</groupId>
             <artifactId>liquibase-postgresql</artifactId>
-            <version>3.0</version>
+            <version>4.3.0</version>
```
[the remark this answers is not in the document]
Comment (kavitha-sundararajan) [reply hip-atomfeed/pom.xml]: refactored to same version

File: hip-atomfeed/pom.xml [hunk 4]
```diff
@@ -104,27 +117,10 @@
...
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-annotations</artifactId>
-            <version>2.9.10</version>
-        </dependency>
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <version>3.4.0</version>
+            <version>4.9.0</version>
```
[the remark this answers is not in the document]
Comment (kavitha-sundararajan) [reply hip-atomfeed/pom.xml]: refactored all versions in the pom as properties

**Review by sivareddytw — APPROVED** [review]

**Pull request merged**